px-tooltip.html

```diff
@@ -852,13 +922,21 @@
     _getClass: function() {
       var res = 'tooltip-container ';
 
-      if(!this._isShowing) {
+      if(!this.visible) {
         res += ' hidden ';
       }
 
       //style the arrow
       res += this._tooltipPos;
       return res;
+    },
+
+    _ignoreTargetEventsChanged: function() {
+
+      //make sure the handlers are removed
```

Comment:
This function handles the case: when `ignoreTargetEvents` changes from `false` to `true`, stop listening to events on the target.

Doesn't it also need to handle the case: when `ignoreTargetEvents` changes from `true` to `false`, start listening to events on the target?

Reply:
fair point 👍